dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/config_color_theme.tsx

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useCallback } from 'react';
+import {
+    EuiButton,
+    EuiAccordion,
+    EuiFormRow,
+    EuiColorPicker,
+    EuiSpacer,
+    EuiIcon,
+    EuiFlexGroup,
+    EuiFlexItem,
+    htmlIdGenerator,
+    EuiComboBox,
+} from '@elastic/eui';
+import { isEmpty } from 'lodash';
+
+
+export const ConfigColorTheme = ({
+    visualizations,
+    schemas,
+    vizState = [],
+    handleConfigChange,
+    sectionName = 'Color Theme',
+}: any) => {
+    const { data } = visualizations;
+    const { data: vizData = {}, metadata: { fields = [] } = {} } = data?.rawVizData;
+
+    const addButtonText = '+ Add color theme';
```

Comment:
if this is just a constant, can you declare it in constants file?

Reply:
Sure @kavithacm !